easistrain/EDD/calibrationEDD.py

```diff
@@ -223,12 +223,12 @@ def calibEdd(
     infoGroup.create_dataset(
         "scanNumberHorizontalDetector",
         dtype=h5py.string_dtype(encoding="utf-8"),
-        data=scanNumberHorizontalDetector,
+        data=str(scanNumberHorizontalDetector),
     )  ## save of the number of the scan containing the calibration pattern of the horizontal detector in infos group
     infoGroup.create_dataset(
         "scanNumberVerticalDetector",
         dtype=h5py.string_dtype(encoding="utf-8"),
-        data=scanNumberVerticalDetector,
+        data=str(scanNumberVerticalDetector),
```

Comment:
This is to support integers for `scanNumberVerticalDetector` (same for `scanNumberHorizontalDetector`). Previously, putting `1` instead of `'1'` made the program crash